modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Posts/Detail.cshtml

```diff
@@ -57,14 +57,14 @@
                                                 <i class="fa fa-comment"></i> @L["CommentWithCount", @Model.CommentCount]
 
 
-                                                @if (await Authorization.IsGrantedAsync(BloggingPermissions.Posts.Update))
+                                                @if (await Authorization.IsGrantedAsync(BloggingPermissions.Posts.Update) || (CurrentUser.Id.HasValue && CurrentUser.Id == Model.Post.CreatorId))
                                                 {
                                                     <span class="seperator">|</span>
                                                     <a asp-page="./Edit" asp-route-postId="@Model.Post.Id" asp-route-blogShortName="@Model.BlogShortName">
                                                         <i class="fa fa-pencil"></i> @L["Edit"]
                                                     </a>
                                                 }
-                                                @if (await Authorization.IsGrantedAsync(BloggingPermissions.Posts.Delete) || (CurrentUser.Id.HasValue && CurrentUser.Id == Model.Post.CreatorId))
+                                                @if (await Authorization.IsGrantedAsync(BloggingPermissions.Posts.Delete))
                                                 {
```

Comment:
Users are not allowed to delete their own Posts?
https://github.com/abpframework/abp/blob/b0adb9ff09e8fe1db495adf06cf544478f7daf68/modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Posts/PostAuthorizationHandler.cs#L38

Reply:
> Users are not allowed to delete their own Posts?
> https://github.com/abpframework/abp/blob/b0adb9ff09e8fe1db495adf06cf544478f7daf68/modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Posts/PostAuthorizationHandler.cs#L38

Sorry, my branch missed this commit:
https://github.com/abpframework/abp/commit/b0adb9ff09e8fe1db495adf06cf544478f7daf68#diff-e461a1b98fc07088163606c19001917c

Reply:
You can merge `dev `into your branch.